src/serlio/util/MayaUtilities.h

```diff
@@ -22,7 +22,7 @@ void statusCheck(MStatus status, const char* file, int line);
 
 template<typename F>
 void forAllAttributes(const MFnDependencyNode& node, F func) {
-	for (size_t i = 0; i < node.attributeCount(); i++) {
+	for (unsigned int i = 0; i < node.attributeCount(); i++) {
```

Comment:
maybe best to use 'auto' when using types dictated by thirdparty API?

Reply:
I don't understand what you mean here. I can't use `auto` to declare `i`.

Reply:
true, would need decltype... ignore...